gemrb/core/Interface.cpp

```diff
@@ -1205,7 +1205,7 @@ int Interface::Init(InterfaceConfig* cfg)
 
 	Log(MESSAGE, "Core", "GemRB Core Initialization...");
 	Log(MESSAGE, "Core", "Initializing Video Driver...");
-	video = Holder<Video>(( Video * ) PluginMgr::Get()->GetDriver(&Video::ID, config.VideoDriverName.c_str()));
+	video = std::shared_ptr<Video>(static_cast<Video*>(PluginMgr::Get()->GetDriver(&Video::ID, config.VideoDriverName.c_str())));
```

Comment:
we should be using `static_ptr_cast` instead

Reply:
You mean like?
```cpp
std::shared_pointer_cast<Video>(std::shared_ptr<Plugin>(PluginMgr...));
```
I just wonder whether this makes any difference here.